src/In.ProjectEKA.HipService/Discovery/Patient/Ranker/FirstNameRanker.cs

```diff
@@ -0,0 +1,22 @@
+namespace In.ProjectEKA.HipService.Discovery.Patient.Ranker
+{
+    using Helper;
+    using HipLibrary.Patient.Model.Response;
+    using Matcher;
+    using Patient = Model.Patient;
+
+    public class FirstNameRanker : IRanker<Patient>
+    {
+        public PatientWithRank<Patient> Rank(Patient patient, string firstName)
+        {
+            var diff = FuzzyNameMatcher.LevenshteinDistance(patient.FirstName, firstName);
+            if (diff == 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)